CITATION.cff

```diff
@@ -0,0 +1,17 @@
+cff-version: 1.2.0
+title: >-
+  $\texttt{Spglib}$: a software library for crystal symmetry
+  search
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Atsushi
+    family-names: Togo
+  - given-names: Isao
+    family-names: Tanaka
+identifiers:
+  - type: doi
+    value: 10.48550/arXiv.1808.01590
```

Comment:
The generated Bibtex does not seem to pull in this info:
https://github.com/lan496/spglib/tree/how-to-cite
```biblatex
@software{
Togo_texttt_Spglib_a_software,
author = {Togo, Atsushi and Tanaka, Isao},
title = {{$\texttt{Spglib}$: a software library for crystal symmetry search}},
url = {https://github.com/spglib/spglib}}
```